lib/active_model/serializer/adapter/base.rb

```diff
@@ -0,0 +1,7 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      Base = ActiveModelSerializers::Adapter::Base
```

Comment:
Maybe leave the warnings out for the moment?  Does delegate class not allow inheritance? Maybe simple delegator?